code/modules/mob/living/carbon/human/species.dm

```diff
@@ -105,6 +105,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	var/datum/action/innate/flight/fly
 	///The icon used for the wings
 	var/wings_icon = "Angel"
+	///Used to determine what description to give when using a potion of flight, if null it will describe them as growing new wings
+	var/HAS_INNATE_WINGS
```

Comment:
Use lowercase for variable.
Either `	var/has_inate_wings = FALSE` or make a bitflag-var with a `HAS_INNATE_WINGS` define.